src/test/java/api/loans/LoanAPITests.java

```diff
@@ -1582,6 +1596,24 @@ public void canCreateMultipleLoansWithServicePoints()
 
   }
 
+  private void loanHasExpectedProperties(JsonObject loan, IndividualResource user) {
+    loanHasExpectedProperties(loan);
+
+    if (user == null) {
+      return;
+    }
+
+    JsonObject borrower = loan.getJsonObject(LoanProperties.BORROWER);
+    JsonObject personalInfo = user.getJson().getJsonObject("personal");
+
+    hasProperty(LoanProperties.BORROWER, loan, "loan");
+    hasProperty("firstName", borrower, "borrower", personalInfo.getString("firstName"));
```

Comment:
Ok, I had in my mind explicit examples in the specific tests. I guess this is ok.